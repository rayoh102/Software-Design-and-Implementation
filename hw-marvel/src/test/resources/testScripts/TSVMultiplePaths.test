# Tests if, given multiple paths, we return the shortest one

# Load Graph
LoadGraph custom sampleData.tsv

# Find path (shortest should be single hop, longer one would be three hops)
FindPath custom Hero1 Hero5