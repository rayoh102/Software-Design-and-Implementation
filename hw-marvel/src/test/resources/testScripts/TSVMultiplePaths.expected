# Tests if, given multiple paths, we return the shortest one

# Load Graph
loaded Graph custom

# Find path (shortest should be single hop, longer one would be three hops)
path from Hero1 to Hero5:
Hero1 to Hero5 via C2
